monitors/archive_monitor/isis_archive_monitor_win_service.py

```diff
@@ -0,0 +1,61 @@
+"""
+Holds the class for overidding QueueService framework
+"""
+# Can't import on travis (linux) server
+# pylint:disable=import-error
```

Comment:
Could we do something like:
```
import os
if os.system() == "nt":
    return
```

So that the imports below are only attempted on Windows?